frontend/packages/kubevirt-plugin/src/selectors/vm-template/combined-dependent.ts

```diff
@@ -34,22 +34,13 @@ export const getFlavorLabel = (flavor: string) => {
   return undefined;
 };
 
-export const getOperatingSystems = (
-  templates: TemplateKind[],
-  { workload, flavor, userTemplate }: { workload: string; flavor: string; userTemplate: string },
-) => {
-  let templatesWithLabels;
+export const getOperatingSystems = (templates: TemplateKind[], userTemplate: string) => {
   if (userTemplate) {
-    templatesWithLabels = [
+    return getTemplateOperatingSystems([
       getTemplatesOfLabelType(templates, TEMPLATE_TYPE_VM).find((t) => getName(t) === userTemplate),
-    ];
-  } else {
-    templatesWithLabels = getTemplatesWithLabels(
-      getTemplatesOfLabelType(templates, TEMPLATE_TYPE_BASE),
-      [getWorkloadLabel(workload), getFlavorLabel(flavor)],
-    );
+    ]);
```

Comment:
why? 
this will cause to return the wrong OSs not reflecting the workload and flavor selection.

Reply:
https://bugzilla.redhat.com/show_bug.cgi?id=1868426

Reply:
@suomiy hi, 
from UX side this is wrong to change the selection options of a field by using another field below it.

from a user perspective a common problem this can fix is:
a - a user selects an OS then selects a Flavor or Workload
b - a user want to fix their selection of OS, but the correct OS is missing from the list.

the fix:
a - a user select and OS
b - the user select some flavor
c - all OS options are presented and they can currect the OS selection.

Reply:
this is not about UX, but about correct template selection. The user should be aware of comon templates and create his own if such template is missing (based on os/flavor etc.)

disscussing more in BZ

Reply:
Note:
in the BZ discussion we decided that a viable fix for the problem wil be to:

> 1: override the user selection of flavor/workload upon OS selection

Ido, can we override the values to "null" (or what ever the default value is) ?
this will make sure users know that the values where changed ?

Reply:
yeah good idea